### PR TITLE
Added missing marks

### PR DESCRIPTION
**Pull request by KeyboardRage**
Creates handling of super- and sub-script markings. If sup/sub are used in the Rich Editor in Contentful, it'll crash the entire render  😕

### CODE REVIEW

**Comment by KeyboardRage** [comment]
Not quite sure how to set up the test for Asset though, since it can be any type of file.

Furthermore I haven't yet had time to figure out how it resolves the asset information from the document, it seems all it provides is the asset ID.

File: src/index.js [hunk 1]
```diff
@@ -64,6 +66,25 @@ const defaultNodeRenderers = {
...
+    if (attrs.contentType.startsWith("image/")) {
+      return h("img", {
+        key,
+        src: attrs.url,
+        alt: attrs.title,
+        title: attrs.title,
+        name: attrs.fileName
+      }, next(node.content, key, next));
+    }
+  },
```
Comment (tolgap) [inline src/index.js]: Could you please split this to it's own PR 👍. I have some reservations about merging opinionated stuff as default.

**Review by tolgap — CHANGES_REQUESTED** [review]
@KeyboardRage The `MARKS.SUPERSCRIPT` and `MARKS.SUBSCRIPT` changes are great! Thank you for taking the time.

Could you please remove the addition of `BLOCKS.EMBEDDED_ASSET` from this PR, and perhaps open another PR for it? We can then discuss the implementation of this block separately 🙂.

**Review by tolgap — APPROVED** [review]
LGTM! Thank you @KeyboardRage. I'll release this soon.

**Comment by tolgap** [comment]
@KeyboardRage released as `v3.1.0`! I also added a constraint for `@contentful/rich-text-types: ^15.14.1` where the `SUB` and `SUPER` marks were introduced.
